tests/analyze-fixtures/ArrayDuplicateKeys.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Tests\Compiling\Statements;
+
+class ArrayDuplicateKeys
+{
+    /**
+     * @return array
+     */
+    public function method()
+    {
+        return [
+            'foo' => 'bar',
+            'baz' => 'biz',
+            'foo' => 'joe',
+            0 => 42,
+            1 => 43,
+            0 => 'lala'
+        ];
+    }
+}
```

Comment:
Need some fake data without errors in another methods
To check that Analyzer working okey